src/lib/Constants.ts

```diff
@@ -21,9 +21,16 @@ export const ADDRESSES = {
   ZERO: '0x0000000000000000000000000000000000000000',
   DYDX_USDC_ADDRESS_ROPSTEN: '0x8707a5bf4c2842d46b31a405ba41b858c0f876c4',
   USDC_ADDRESS_MAINNET: '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48',
+  ZERO_EX_EXCHANGE_ADDRESS_ROPSTEN: '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48',
+  ZERO_EX_EXCHANGE_ADDRESS_MAINNET: '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48',
```

Comment:
Add a todo in the code, not just the pr, if something is incorrect/temporary/todo